aws-lc-rs/src/hkdf.rs

```diff
@@ -92,6 +92,7 @@ pub struct Salt {
     key_len: usize,
 }
 
+#[allow(clippy::missing_fields_in_debug)]
```

Comment:
nit: This seems unrelated to this PR. In the future, it would be easier to review with more focused PRs.

Reply:
The latest commit merged changes (like this one) which were already on main, so some of the "noise" like this should be gone from the diff.